frontend/src/components/Modal/RiderModalInfo.tsx

```diff
@@ -1,112 +1,157 @@
+          {errors.netid && <p className={styles.error}>enter a valid netid</p>}
         </div>
-        <div className={cn(styles.gridR2, styles.gridC2)}>
+        <div className={cn(styles.gridR1, styles.gridCSmall3)}>
           <Input
             name="phoneNumber"
             type="text"
             placeholder="Phone Number"
             ref={register({ required: true, pattern: /^[0-9]{10}$/ })}
+            className={styles.firstRow}
           />
-          {errors.phoneNumber && 'enter a valid phone number'}
+          {errors.phoneNumber && (
+            <p className={styles.error}>enter a valid phone number</p>
+          )}
         </div>
-        <div className={cn(styles.gridR3, styles.gridC1)}>
+        <div className={cn(styles.gridR2, styles.gridCBig1)}>
           <Input
             name="needs"
             type="text"
             placeholder="Needs"
-            ref={register({ required: true })}
+            ref={register({
+              validate: (needs) => {
+                if (needs === "") {
+                  return true;
+                }
+                const needsArr = needs.split(",").map((n: string) => n.trim());
+                const isValidNeed = (
+                  acc: boolean, 
+                  val: Accessibility
+                ) => acc && Object.values(Accessibility).includes(val);
+                return needsArr.reduce(isValidNeed, true);
+              },
+            })}
           />
-          {errors.email && 'enter some needs'}
+          {errors.needs?.type === "validate" && (
+            <p className={styles.error}>
+              Invalid needs. You can enter 'Assistant', 'Crutches', or
+              'Wheelchair'
+            </p>
+          )}
         </div>
-        <div className={cn(styles.gridR3, styles.gridC2)}>
+        <div className={cn(styles.gridR2, styles.gridCBig2)}>
           <Input
             name="address"
             type="text"
             placeholder="Address"
-            ref={register({ required: true })}
+            ref={register({
+              required: true,
+              pattern: /^[a-zA-Z0-9\s,.'-]{3,}$/,
+            })}
           />
-          {errors.email && 'enter an address'}
+          {errors.address && (
+            <p className={styles.error}>Please enter an address</p>
+          )}
         </div>
-        <div className={cn(styles.gridR4, styles.gridC1)}>
-          <Label htmlFor="start">Start Date:</Label>
-          <Input
-            type="date"
-            name="start"
-            ref={register({ required: true })}
-          />
-        </div>
-        <div className={cn(styles.gridR4, styles.gridC2)}>
-          <Label htmlFor="end">End Date:</Label>
-          <Input
-            type="date"
-            name="end"
-            ref={register({ required: true })}
-          />
+        <div className={cn(styles.gridR3, styles.gridCAll)}>
+          <p>Duration</p>
+          <div style={{ display: "flex" }}>
```

Comment:
In general, using inline styling isn't really great, just because it makes it harder to keep track of things/update/make accessible or work with media queries (the last part being the key detail here for the future of admin web). Can you move the inline styling to the css file/use classes instead?